scrapers_library/data_portals/muckrock/muckrock_tools/generate_detailed_muckrock_csv.py

```diff
@@ -50,38 +55,40 @@ def get_jurisdiction(jurisdiction_id):
             return ""
     else:
         print("Jurisdiction ID not found in item")
-    
 
+
+output_csv = re.sub(r'_(?=[^.]*$)', '-', args.json_file[:-5]) + '.csv'
 # Open a CSV file for writing
-with open('detailed-muckrock-data.csv', 'w', newline='') as csvfile:
+with open(output_csv, 'w', newline='') as csvfile:
```

Comment:
Nice catch with converting the csv to a variable rather than a magic string!

Given how complex the regex for determining the file name is, my spidy senses tell me we'd likely want to extract this to a separate file and include a docstring describing what it does 🕷️

Reply:
Great idea! I had to fiddle with that regex for like 10 minutes originally...